docs/reference/platforms/index.rst

```diff
@@ -23,48 +23,60 @@ Platform support
 +-----+-------------------------------------+
 
 
-.. |Gradle| replace:: **Gradle**
```

Comment:
curious....did you intentionally use non-breaking spaces for these? I feel like you did and hence, I maintained them. Although, without them, the whole table does fit better on the page.